src/login-form2.php

```diff
@@ -0,0 +1,30 @@
+<?php
+error_reporting(E_ERROR | E_WARNING | E_PARSE | E_NOTICE);  
+ini_set('display_errors' , 1);
+
+include ("account.php");
+
+$db = mysqli_connect($hostname, $username, $password, $project);
+
+if (mysqli_connect_errno())
+  {
+	  echo "Failed to connect to MySQL: " . mysqli_connect_error();
+	  exit();
+  }
+
+mysqli_select_db( $db, "ssc3" ); 
+
+$u = $_POST["user"];
+$p = $_POST["pass"];
+

```

Comment:
missing hash password conversion, will modify later

Reply:
oh ok so just do that and then we'll merge